tfe/data_source_slug.go

```diff
@@ -0,0 +1,67 @@
+package tfe
+
+import (
+	"bytes"
+	"crypto/sha256"
+	"encoding/hex"
+	"fmt"
+	"log"
+	"os"
+
+	slug "github.com/hashicorp/go-slug"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+func dataSourceTFESlug() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceTFESlugRead,
+
+		Schema: map[string]*schema.Schema{
+			"source_path": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+
+			"checksum": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
```

Comment:
Thats a good point. I initially had this because the intent was the user would explicitly call this in the old approach. But now that we are just passing the slug entirely as a map, we can internally look to the "id" and not the "checksum". 

Will remove